Uno/NugetPackageExplorer.Legacy/Views/Behaviors/TreeViewBehavior.cs

```diff
@@ -168,5 +194,43 @@ void ToggleExpansion(TreeView sender, TreeViewItemInvokedEventArgs args)
                 }
             }
         }
+
+        [SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "managed via DoubleClickCommandDisposable")]
+        private static void OnDoubleClickCommandChanged(TreeView control, DependencyPropertyChangedEventArgs e)
+        {
+            GetDoubleClickCommandDisposable(control)?.Dispose();
+            if (e.NewValue is ICommand command)
+            {
+                var subscriptions = new CompositeDisposable();
+
+                var doubleTapped = Observable
```

Comment:
For L204~234, you can skip the publish + connect, and the CompositeDisposable. They are used in [ListViewExtensions.cs#L115](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/blob/4e33207ee9517ecccf1fccd839ab2bab9c21917e/Uno/NugetPackageExplorer.Legacy/Views/Extensions/ListViewExtensions.cs#L115) to avoid double subscriptions. But, it is not needed here.

```cs
var subscription = Observable
    .FromEventPattern<>(...)
    .Select(...)
    .Subscribe(...);
SetDoubleClickCommandDisposable(control, subscription);
```